gatsby-config.js

```diff
@@ -250,10 +251,6 @@ module.exports = {
             matchPath: '/:lang?/career/:id/',
             getLanguageFromPath: true,
           },
-          {
-            matchPath: '/:lang?/career/',
-            getLanguageFromPath: true,
-          },
```

Comment:
This prevented i18n from creating the career page duplicates internally with the correct language. It took a few moments to realize is hence this comment.